oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/blob/RDBBlobStoreTest.java

```diff
@@ -146,7 +146,7 @@ public void testBigBlob() throws Exception {
 
         LOG.info("max blob length for " + blobStoreName + " was " + test);
 
-        int expected = Math.max(blobStore.getBlockSize(), 2 * 1024 * 1024);
+        int expected = Math.max(blobStore.getBlockSize(), 1 * 1024 * 1024);
```

Comment:
I don't recall whether RDBBlobStore requires bigger blobs. In which case, changing the test here would hide regressions with other databases. It might be better to disable this test case just for h2db.

Reply:
I agree with @reschke. This seems to be a limitation of the newer H2 version. The default block size of AbstractBlobStore is 2MB. I assume that's why we have this test.
https://github.com/apache/jackrabbit-oak/blob/c6ddcc55bee3de915459af01e91edad32d538f3d/oak-blob/src/main/java/org/apache/jackrabbit/oak/spi/blob/AbstractBlobStore.java#L120

Reply:
Yes it's a limitation of a new version of H2... An in-memory object contains the array of bytes, and the size of such objects is now limited to 1 MB (I didn't know btw... just read that in the source code)

Reply:
Why does the test consider `blobStore.getBlockSize()` when it is set to 128 in `setUp()`?

Reply:
I reverted this change in https://github.com/apache/jackrabbit-oak/pull/466/commits/4f76f50f7b6c12c3f844772d6c1650a45067a0d0. Instead the test is skipped when it runs on H2.

Reply:
I honestly do not recall.